### ISSUE TITLE
Can't  build kf5-kdelibs4support on MacOS 10.13.1

### ISSUE THREAD

**Issue by azevedoph** [KDE-mac/homebrew-kde#176]
All goes fine until almost the very end, when I get this:

`[ 87%] Automatic MOC for target kdebugdialog5
cd /tmp/kf5-kdelibs4support-20171122-69986-6ym8ek/kdelibs4support-5.40.0/build/src/kioslave/metainfo && /usr/local/Cellar/cmake/3.10.0/bin/cmake -E cmake_autogen /tmp/kf5-kdelibs4support-20171122-69986-6$
/usr/local/Cellar/cmake/3.10.0/bin/cmake -E cmake_autogen /tmp/kf5-kdelibs4support-20171122-69986-6ym8ek/kdelibs4support-5.40.0/build/CMakeFiles/kf5-config_autogen.dir Release
cd /tmp/kf5-kdelibs4support-20171122-69986-6ym8ek/kdelibs4support-5.40.0/build/src/kdebugdialog && /usr/local/Cellar/cmake/3.10.0/bin/cmake -E cmake_autogen /tmp/kf5-kdelibs4support-20171122-69986-6ym8ek$
Warning: Could not locate service type file kservicetypes5/kdedmodule.desktop, tried ("/Users/paulo/Library/Application Support", "/Library/Application Support") and ":/kservicetypes5/kdedmodule.desktop"$
About to parse service type file "kdedmodule.desktop"
[ 87%] Built target kf5-config_autogen
/Applications/Xcode.app/Contents/Developer/usr/bin/make -f CMakeFiles/kf5-config.dir/build.make CMakeFiles/kf5-config.dir/depend
make[2]: *** [src/solid-networkstatus/kded/networkstatus.json] Error 1
make[1]: *** [src/solid-networkstatus/kded/CMakeFiles/kded_networkstatus_autogen.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
cd /tmp/kf5-kdelibs4support-20171122-69986-6ym8ek/kdelibs4support-5.40.0/build && /usr/local/Cellar/cmake/3.10.0/bin/cmake -E cmake_depends "Unix Makefiles" /tmp/kf5-kdelibs4support-20171122-69986-6ym8ek$
[ 87%] Built target kf5deprecatedwidgets_autogen
[ 87%] Built target kio_metainfo_autogen
Scanning dependencies of target kf5-config
/Applications/Xcode.app/Contents/Developer/usr/bin/make -f CMakeFiles/kf5-config.dir/build.make CMakeFiles/kf5-config.dir/build
[ 87%] Building CXX object CMakeFiles/kf5-config.dir/kde-config.cpp.o
[ 88%] Building CXX object CMakeFiles/kf5-config.dir/kf5-config_autogen/mocs_compilation.cpp.o
/usr/local/Homebrew/Library/Homebrew/shims/super/clang++  -DKCOREADDONS_LIB -DKDELIBS4SUPPORT_DEPRECATED="" -DKDELIBS4SUPPORT_DEPRECATED_EXPORT=KDELIBS4SUPPORT_EXPORT -DKDELIBS4SUPPORT_DEPRECATED_NO_EXPO$
/usr/local/Homebrew/Library/Homebrew/shims/super/clang++  -DKCOREADDONS_LIB -DKDELIBS4SUPPORT_DEPRECATED="" -DKDELIBS4SUPPORT_DEPRECATED_EXPORT=KDELIBS4SUPPORT_EXPORT -DKDELIBS4SUPPORT_DEPRECATED_NO_EXPO$
[ 88%] Built target kdebugdialog5_autogen
[ 88%] Linking CXX executable bin/kf5-config
/usr/local/Cellar/cmake/3.10.0/bin/cmake -E cmake_link_script CMakeFiles/kf5-config.dir/link.txt --verbose=1
/usr/local/Homebrew/Library/Homebrew/shims/super/clang++   -std=c++0x -fno-operator-names -fno-exceptions -Wno-gnu-zero-variadic-macro-arguments -Wall -Wextra -Wcast-align -Wchar-subscripts -Wformat-secu$
[ 88%] Built target kf5-config
make: *** [all] Error 2`

Couldn't really find anything related to it on the issues, either open or closed. I figure it's something to do with Solid, from those lines below, but can't say what:
`make[2]: *** [src/solid-networkstatus/kded/networkstatus.json] Error 1
make[1]: *** [src/solid-networkstatus/kded/CMakeFiles/kded_networkstatus_autogen.dir/all] Error 2`

Any help would be very much appreciated! Thank you!

**Comment by sl1pkn07** [comment]
~~~
Could not locate service type file kservicetypes5/kdedmodule.desktop,  tried ("/Users/paulo/Library/Application Support", "/Library/Application Support") and ":/kservicetypes5/kdedmodule.desktop"
~~~

that file is part of kf5-kded

~~~
└───╼  brew list kf5-kded | grep kdedmodule.desktop
/usr/local/Cellar/kf5-kded/5.40.0_1/share/kservicetypes5/kdedmodule.desktop
~~~

but for my fault, this is not point properly in the kf5-kded caveats
this is fixed now in d3ec9c051863a3b26d3011798c9e04b0e3485d36


greetings and sorry

**Comment by azevedoph** [comment]
Thank you! It worked now =]
